compiler/dalgona/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.0)
+project(dalgona)
```

Comment:
Is this necessary here? (I'm also not good at cmake but this `procject` appeared in top cmake file...)

Reply:
> Is this necessary here? 

I don't think so. I removed the code. Thanks.